src/pal/inc/unixasmmacros.inc

```diff
@@ -5,23 +5,23 @@
 #define INVALIDGCVALUE 0xCCCCCCCD
 
 #if defined(__APPLE__)
-#define C_FUNC(name) _##name
-#define EXTERNAL_C_FUNC(name) C_FUNC(name)
-#define LOCAL_LABEL(name) L##name
+#define C_FUNC(_name) _##_name
```

Comment:
I am not sure I understand this change, as it is effectively no-op

Reply:
I just did a search and replace in the file for name with _name per your recommendation. Did I misunderstand again?

Reply:
I removed the last rename now. It might be easier if you did the edit on the file.

Reply:
I am sorry for not being clear. I was referring to the ARM64 case only - I was trying to say that instead of changing the name@PLT to _name as you did, I think it should be changing it to name. I was referring to the created symbol name, not to the variable name.